src/SourceBuild/Arcade/eng/common/templates/job/source-build-build-tarball.yml

```diff
@@ -0,0 +1,203 @@
+# Builds a source-build tarball
```

Comment:
The majority of this content comes from [src/SourceBuild/Arcade/eng/common/templates/steps/source-build-build-tarball.yml](https://github.com/dotnet/installer/pull/13500/files#diff-19d4b26a4d7e8be7c92961a9873eeebc6716cc6872a48f976a3656ade69f9c2c).